src/practical_settings/im.tex

```diff
@@ -31,30 +31,46 @@ \subsection{ejabberd}
 
 \subsubsection{Tested with Versions}
 \begin{itemize*}
-  \item Debian Wheezy 2.1.10-4+deb7u1
+  \item 13.6 - 14.12
 \end{itemize*}
 
 \subsubsection{Settings}
 ejabberd is one of the popular Jabber servers. In order to be compliant
 with the manifesto, you should adapt your configuration\footnote{\url{http://www.process-one.net/docs/ejabberd/guide_en.html}}:
 
-\configfile{ejabberd.cfg}{108-109,111-111,120-126,172-172,179-179,184-184}{%
+\configfile{ejabberd.yml}{104-107,113-113,114-114,119-119,123-123,125-125,127-127,135-137,141-143,192-192,200-200,204-206}{%
   TLS setup for ejabberd}
 % 
 \subsubsection{Additional settings}
-Older versions of ejabberd ($ < $ 2.0.0) need to be patched\footnote{\url{http://hyperstruct.net/2007/06/20/installing-the-startcom-ssl-certificate-in-ejabberd/}} to be able to parse all of the certificates in the CA chain.
+It is possible to explicitly specify a cipher string for TLS connections.
+\configfile{ejabberd.yml}{104-107,113-114,119-119,123-123,125-125,127-127,134-137,139-139,141-143,192-192,200-200,204-207,211-211}{Specifying a cipher order and enforcing it}
 
-Newer versions of ejabberd now support specifying the cipher string in the config file. See the commit message: \url{https://github.com/processone/ejabberd/commit/1dd94ac0d06822daa8c394ea2da20d91c8209124}. However, this change did not yet make it into the stable release at the time of this writing. 
+Note that we are setting the SSL option cipher\_server\_preference. This enforces our cipher order when negotiating which ciphers are used, as the cipher order of some clients chooses weak ciphers over stronger ciphers.\footnote{\url{https://blog.thijsalkema.de/me/blog//blog/2013/09/02/the-state-of-tls-on-xmpp-3/}}
 
+Other options you may want to take a look at are:
+\begin{itemize}
+\item The log level logs all login, including username and IP address, by default. Change the \texttt{loglevel} setting to avoid this:\\
+\configfile{ejabberd.yml}{31-39}{Changing the logging behaviour}
```

Comment:
I opted to include it because I believe in minimizing the amount of data an attacker can get if he/she compromises the server, but I'll leave it to the editors to decide if they want to have this included or not. 

The same goes for the open registration: The default may be a bit counterintuitive, so I'd rather err on the side of caution and mention it, but if the maintainers / editors want it out, I'll take it out.